kafka-rest/pom.xml

```diff
@@ -178,6 +178,14 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Exclude from checkstyle checks, as style in different in Jetty from where these are taken.

Reply:
remove them